examples/feast_feature_store/config.yaml

```diff
@@ -0,0 +1,10 @@
+name: feast_pipeline
```

Comment:
For other integrations you mentioned to run via `-c config.yaml`, why not for feast?